package.json

```diff
@@ -33,12 +34,13 @@
     "typescript": "^4.2.3"
   },
   "scripts": {
-    "install": "cd packages/docs && npm ci",
+    "install": "test -n $SKIP_DOCS_INSTALL || (cd packages/docs && npm ci)",
     "rebuild": "npm run clean && npm run build",
     "build": "lerna run --stream build",
     "build.watch": "lerna run --stream build.watch",
     "test": "lerna run --stream test",
     "test.watch": "lerna run --stream test.watch",
+    "ci-load": "node ./packages/test/lib/perf/all-in-one.js --server-address $TEMPORAL_TESTING_SERVER_URL",
```

Comment:
Why don't we call this `perf` instead of `load`?
I'd keep this consistent, AFAIC, we can rename either the test files or the script name.